### PR TITLE
Skip redundant apex bridge e2e tests when env variable not set

### PR DESCRIPTION
**Pull request by babinskizoltan**
# Description

Please provide a detailed description of what was done in this PR

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it

# Checklist

- [ ] I have assigned this PR to myself
- [ ] I have added at least 1 reviewer
- [ ] I have added the relevant labels
- [ ] I have updated the official documentation
- [ ] I have added sufficient documentation in code

## Testing

- [ ] I have tested this code with the official test suite
- [ ] I have tested this code manually

### Manual tests

Please complete this section if you ran manual tests for this functionality, otherwise delete it

# Documentation update

Please link the documentation update PR in this section if it's present, otherwise delete it

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it

### CODE REVIEW

**Comment by babinskizoltan** [comment]
"SKIP_E2E_REDUNDANT_TESTS" env variable should be set to "true" only for PRs. For nightly and everywhere else should be false

**Review by igorcrevar — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -83,7 +83,7 @@ test-e2e-polybft: check-go
 .PHONY: test-e2e-apex-bridge
 test-e2e-apex-bridge: check-go
 	go build -o artifacts/blade .
-	env EDGE_BINARY=${PWD}/artifacts/blade E2E_TESTS=true E2E_LOGS=true \
+	env EDGE_BINARY=${PWD}/artifacts/blade E2E_TESTS=true E2E_LOGS=true SKIP_E2E_REDUNDANT_TESTS=true \
```
Comment (babinskizoltan) [inline Makefile]: branislav should revert this and pass the env variable through .yaml files

**Comment by babinskizoltan** [comment]
Consider also disabling polybft tests for PRs

**Review by igorcrevar — APPROVED** [review]
Approved, just fix environment variable setting

**Pull request merged**